src/IonBinaryWriter.ts

```diff
@@ -375,7 +375,7 @@ export class BinaryWriter extends AbstractWriter {
   private checkWriteValue() : void {
     this.checkClosed();
     if (this.state === States.STRUCT_FIELD) {
-      throw new Error("Expected a struct field name instead of a value");
+      throw new Error("Expected a struct field name instead of a value, Call writeFieldName(string) with the desired name before calling stepIn(IonType) or writeIonType()");
```

Comment:
Nit: The 'c' in "Call" follows a comma and so doesn't need to be capitalized.
> ...instead of a value. Call writeFieldName(string)...

or

> instead of a value, call writeFieldName(string)

would work.